client/components/grid-defaults/grid-defaults.service.js

```diff
@@ -53,6 +55,36 @@ function GridDefaults($timeout, uiGridGroupingConstants, Student,
       visible: false
     };
   };
+  colDefs.iep = function(name) {
+    return {
+      name: name || 'student.iep',
+      displayName: 'IEP',
+      enableCellEdit: true,
+      type: 'boolean',
+      width: 100,
+      treeAggregationType: uiGridGroupingConstants.aggregation.SUM
+    };
+  };
+  colDefs.cfa = function(name) {
+    return {
+      name: name || 'student.cfa',
+      displayName: 'CFA',
+      enableCellEdit: true,
+      type: 'boolean',
```

Comment:
I thought we were trying to disable cell editing, is there a reason why these columns are editable?